abi/0.10.0/Organized.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "constant": true,
+        "inputs": [],
+        "name": "organization",
+        "outputs": [
+            {
+                "name": "",
+                "type": "address"
+            }
+        ],
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
+    },
+    {
+        "inputs": [
+            {
+                "name": "_organization",
+                "type": "address"
+            }
+        ],
+        "payable": false,
+        "stateMutability": "nonpayable",
+        "type": "constructor"
+    }
```

Comment:
We don't need all the ABIs for graph node. I remember we discussed in meeting to write only those ABIs which is needed for facilitator. e.g. Organized ABI is not needed.

Similar comment for other ABIs also.